src/examples/graph.js

```diff
@@ -52,54 +52,63 @@ const sample: IGraph = {
   edges: [
     {
       handleText: '5',
+      tooltipText: '5',
```

Comment:
A good use case is if the tooltip is different than the handleText. Please add an example that includes a more descriptive tooltip for that case.